package.json

```diff
@@ -90,7 +94,7 @@
 		"@wordpress/base-styles": "1.0.0",
 		"@wordpress/browserslist-config": "2.5.0",
 		"@wordpress/compose": "3.5.0",
-		"@wordpress/data": "4.7.0",
+		"@wordpress/data": "4.6.0",
```

Comment:
We were getting 4.6.0 (as a dependency of `@wordpress/compose@3.5.0`) and 4.7.0 (as package.json dependency), thus breaking the `build-extensions` build. This can be bumped up again together with the rest of the `@wordpress/*` packages (https://github.com/Automattic/jetpack/pull/13593).